app/controllers/users_controller.rb

```diff
@@ -1,5 +1,11 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
+    # @users = User.all.order(created_at: :desc)
```

Comment:
- [ ] Please remove the following commented out codes in order to keep your codebase neat 😃